examples/java/spring-boot/src/main/resources/log4j2-spring.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
did we want to set up Spotless' XML formatting for this? Now we've got XML files to manage, it'd be good to keep formatting consistent